src/scheduler_multi_app.cpp

```diff
@@ -95,21 +199,20 @@ void multi_app_sched::schedule(const job_queueT &job_queue, fast::MQTT_communica
 
 		while (true) {
 			// for all host-id of new job
-			auto marked_machines = check_membw(config);
+			const auto marked_machines = check_membw(config);
 
 			// everything fine?
 			if (marked_machines.empty()) break;
 
 			if (controller.update_supported()) {
```

Comment:
Yes, actually we need to that the job (see last commit). Otherwise, we could consider triggering the S/R protocol in freeze/thaw.